app/components/organization/Pipeline/bar.js

```diff
@@ -69,7 +69,11 @@ class Bar extends React.Component {
 
     if (this.props.href) {
       return (
-        <AnchoredPopover position="absolute" style={{ height: '100%', left: this.props.left, bottom: 0 }}>
+        <AnchoredPopover
+          position="absolute"
+          style={{ height: '100%', left: this.props.left, bottom: 0 }}
+          width={300}
```

Comment:
given that both places which use `AnchoredPopover` now have a default of 300, maybe we should update `AnchoredPopover`’s defaults instead?